src/helics/network/NetworkBrokerData.cpp

```diff
@@ -76,7 +76,9 @@ std::shared_ptr<helicsCLI11App>
         },
         "identifier for the broker, this is either the name or network address use --broker_address or --brokername "
         "to explicitly set the network address or name the search for the broker is first by name");
-    nbparser->add_option("--brokername", brokerName, "the name of the broker");
+    nbparser->add_option("--brokername,--brokerName,--broker_name",
```

Comment:
This file has several other instances where ignore underscore/case could be used to make the list of options less verbose.